src/series.jl

```diff
@@ -15,7 +15,7 @@ prepareSeriesData(x) = error("Cannot convert $(typeof(x)) to series data for plo
 prepareSeriesData(::Nothing) = nothing
 prepareSeriesData(t::Tuple{T, T}) where {T<:Number} = t
 prepareSeriesData(f::Function) = f
-prepareSeriesData(a::AbstractArray{<:MaybeNumber}) = replace!(
+prepareSeriesData(a::AbstractArray{<:MaybeNumber}) = replace(
```

Comment:
AFAICS this creates yet another copy of the input data, right? I think this was added because we knew that a copy is already done in `compute_xyz` so it's safe to keep modifying the array?

Reply:
OK, thanks for the info! I'll see if I can fix this without adding another copy.

Reply:
best thing would be to have a test for that

Reply:
the problem might be that copying a StaticVector returns a StaticVector.

Reply:
Yes I think so too